spring-core/src/main/java/org/springframework/core/serializer/DefaultDeserializer.java

```diff
@@ -67,8 +67,7 @@ public DefaultDeserializer(@Nullable ClassLoader classLoader) {
 	@Override
 	@SuppressWarnings("resource")
 	public Object deserialize(InputStream inputStream) throws IOException {
-		ObjectInputStream objectInputStream = new ConfigurableObjectInputStream(inputStream, this.classLoader);
-		try {
+		try (var objectInputStream = new ConfigurableObjectInputStream(inputStream, this.classLoader)){
```

Comment:
We don't use `var` in Spring Framework.

Thus, please remove all usage of `var` in this PR.

Reply:
Good to know, removed all instances.